src/dockerfile.asb.endpoint

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/windows/servercore:1803
```

Comment:
Feels like the docket for should be called something among the lines of what it represents, src/dockerfile.asb.endpointoriented. Having just the `endpoint` part in the name reads as if this was an endpoint file. Don't block this, just sharing good I read it first 🙂

Reply:
Do you have a suggestion that we could use? We need to give the container image a good name too...

I'll merge in the meantime anyway.